src/lib/features/playground/playground-service.ts

```diff
@@ -112,18 +135,22 @@ export class PlaygroundService {
                         projectId: featureProject[feature.name],
                         variant: client.getVariant(feature.name, clientContext),
                         name: feature.name,
+                        environment,
```

Comment:
I'm passing environment and context from input to output because the new UI requires those values